src/client.py

```diff
@@ -0,0 +1,48 @@
+"""Client side of server."""
+
+import socket
+from sys import argv, version_info
+
+
+def client(message):
+    """Setup client side."""
+    infos = socket.getaddrinfo('127.0.0.1', 5016)
+    stream_info = [i for i in infos if i[1] == socket.SOCK_STREAM][0]
+
+    client = socket.socket(*stream_info[:3])
+    client.connect(stream_info[-1])
+    print("Connecting...")
+
+    message += '\r\n'
+
+    if version_info[0] == 2:
+        print('python 2')
+        message = message.decode("utf8")
+
+    print("Sending: ", message.encode('utf8'))
+    client.sendall(message.encode('utf8'))
+
+    buffer_length = 8
+    result = u""
+
+    while result[-2:] != u"\r\n":
+
+        part = client.recv(buffer_length)
+        result += part.decode('utf8')
+
+    client.close()
+    print(result[:-2])
+    return result[:-2]
+
+
+def main():
+    """Function initiates our echo server."""
+    try:
+        script, message = argv
+        client(message)
+    except:
```

Comment:
You can be a lot more explicit in this part. What kind of error are you expecting to except? You should always avoid using a "bare except" whenever you can.